course_info/templates/course_info/partials/people.html

```diff
@@ -261,7 +260,7 @@ <h3 class="breadcrumbs">
 
   <!-- begin course membership interface -->
   <table datatable="" dt-options="dtOptions" dt-columns="dtColumns"
-         dt-instance="dtInstance" class="table table-striped table-hover">
+         dt-instance="dtInstance" class="table table-striped">
```

Comment:
I think the intention was that only the warnings/failures table should not have hover styling; the course membership table should continue to have hover styling, as you can actually perform actions on those (i.e. delete).